schema.graphql

```diff
@@ -68,20 +68,31 @@ type Advertisement {
   createdAt: DateTime!
   creator: User
   endDate: Date!
-  link: String!
+  mediaUrl: URL!
   name: String!
-  orgId: ID!
+  organization: Organization
   startDate: Date!
   type: AdvertisementType!
   updatedAt: DateTime!
 }
 
+type AdvertisementEdge {
+  cursor: String
+  node: Advertisement
+}
+
 enum AdvertisementType {
   BANNER
   MENU
   POPUP
 }
 
+type AdvertisementsConnection {
+  edges: [AdvertisementEdge]
+  pageInfo: ConnectionPageInfo
+  totalCount: Int
+}
```

Comment:
Can you please also update the PR description explaining what AdvertisementEdge and Connection means in layman terms.
Thanks
@disha1202